tests/wpt/web-platform-tests/html/semantics/forms/the-form-element/form-action-url-iframe.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<base href="resources/"></base>
```

Comment:
This file should be inside the `resources` directory itself (otherwise the test harness will think that it is a test). You could move the "target.html" file a subdirectory of `resources` and use that as the new base URL.